help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -110,7 +110,12 @@ To configure details for the destination, fill in the required and optional fiel
   future.
 *  **[!UICONTROL Description]**: A description that will help you identify this
   destination connection/instance in the future.
-*  **[!UICONTROL Name of your source partner]**: The name you would like to go by as a source in Magnite Streaming's platform
+*  **[!UICONTROL Name of your source partner]**: The name you would like to go by as a source in Magnite Streaming's platform. 
+  
+>[!NOTE]
```

Comment:
Please rephrase this as such:


`The source partner name must be an all-lowercase string which matches the name of the Amazon S3 delivery bucket you have configured with Magnite and set up in the [authenticate to destination](#authenticate) step. The supported characters include 'a-z', '0-9', '-'(dash), or '_'(underscore).`

Reply:
Changed the Note due to the new requirement that allows uppercase letters (that later get converted to lowercase). Please review. Thanks!